spec-0004/index.md

```diff
@@ -18,38 +19,99 @@ If relevant, include examples of how the new functionality would be used,
 intended use-cases, and pseudo-code illustrating its use.
 -->
 
-This SPEC recommends that projects test against nightly wheels of several
-widely-used projects in the Scientific Python ecosystem.
+This SPEC describes how to test against nightly wheels of several widely used
+projects and how to create nightly wheels for your project.
 
-There are two main motivations for this recommendation:
+Regularly running your project's tests while using the nightly version of your
+dependencies allows you to spot problems caused by changes before a new release
+is made. This way potential issues can be resolved before they find their way
+into a release, at which point it becomes much harder to change or revert
+something.
```

Comment:
Maybe clarify that it's also/mostly to spot issue upstream and let upstream know. It might not be clear to everyone.